src/foam/blob/Blob.js

```diff
@@ -250,7 +283,7 @@ foam.CLASS({
   ],
   properties: [
     {
-      class: 'String',
+      class: 'Int',
```

Comment:
Id should be a string not an int, the real BlobServices use a hex representation of the SHA256 hash of the blob.  Only in TestBlobService did we use an int, but that was just for convenient, we can convert the ints to strings in TestBlobService.